src/encode.c

```diff
@@ -1057,7 +1057,7 @@ PyImaging_JpegEncoderNew(PyObject *self, PyObject *args) {
 
     if (!PyArg_ParseTuple(
             args,
-            "ss|nnnnnnnnOy#y#y#",
+            "ss|nnnnnnnnOz#y#y#",
```

Comment:
@radarhere what do you think about using a `#z` parameter?  advantages are:

* allows passing `None` directly
* no need to check for whether to `str.encode()`